test/test_fmcomms5_p.py

```diff
@@ -0,0 +1,229 @@
+import pytest
+
+hardware = "fmcomms5"
+classname = "adi.FMComms5"
+
+
+#########################################
+@pytest.mark.parametrize("classname, hardware", [(classname, hardware)])
+@pytest.mark.parametrize(
+    "attr, start, stop, step, tol",
+    [
+        ("tx_hardwaregain", -86, 0.0, 0.25, 0),
+        ("rx_lo", 70000000, 6000000000, 1, 8),
+        ("tx_lo", 47000000, 6000000000, 1, 8),
+        ("sample_rate", 2084000, 61440000, 1, 4),
+    ],
+)
+def test_fmcomms5_attr(
+    test_attribute_single_value, classname, hardware, attr, start, stop, step, tol
+):
+    test_attribute_single_value(classname, hardware, attr, start, stop, step, tol)
+
+
+#########################################
+@pytest.mark.parametrize("classname, hardware", [(classname, hardware)])
+@pytest.mark.parametrize("attr, tol", [("loopback", 0)])
+@pytest.mark.parametrize("val", [0, 1, 2])
+def test_fmcomms5_loopback_attr(
+    test_attribute_single_value_str, classname, hardware, attr, val, tol
+):
+    test_attribute_single_value_str(classname, hardware, attr, val, tol)
+
+
+#########################################
+@pytest.mark.parametrize("classname, hardware", [(classname, hardware)])
+@pytest.mark.parametrize("channel", [0, 1, [0, 1]])
+def test_fmcomms5_rx_data(test_dma_rx, classname, hardware, channel):
+    test_dma_rx(classname, hardware, channel)
+
+
+#########################################
+@pytest.mark.parametrize("classname, hardware", [(classname, hardware)])
+@pytest.mark.parametrize("channel", [0, 1, [0, 1]])
+def test_fmcomms5_tx_data(test_dma_tx, classname, hardware, channel):
+    test_dma_tx(classname, hardware, channel)
+
+
+#########################################
+@pytest.mark.parametrize("classname, hardware", [(classname, hardware)])
+@pytest.mark.parametrize("channel", [0, 1])
+def test_fmcomms5_loopback(test_dma_loopback, classname, hardware, channel):
+    test_dma_loopback(classname, hardware, channel)
+
+
+#########################################
+@pytest.mark.parametrize("classname, hardware", [(classname, hardware)])
+@pytest.mark.parametrize("channel", [0])
+@pytest.mark.parametrize(
+    "param_set",
+    [
+        dict(
+            tx_lo=1000000000,
```

Comment:
Might me useful to set the chip b LOs to somewhere away from this frequencies or you can get images from those transmitters